### DISCUSSION TITLE
Set legend location

### DISCUSSION THREAD

**Discussion by rgcoe** [pydata/xarray#6325] [Q&A] [answered]
Is there a way to set the legend location when plotting, e.g., 

```python
ds.my_var.plot(legend_kwargs=dict(loc=1))
```

**Comment by mathause** [comment, marked as answer]
I don't think there is a keyword. Maybe you can use [seaborn.move_legend](https://seaborn.pydata.org/generated/seaborn.move_legend.html?highlight=legend#seaborn.move_legend)?

**Reply by rgcoe** [reply]
That's super helpful, thanks @mathause! For now this is great, but perhaps the `xarray` developers might consider adding an argument for the `plot` methods to take care of this in a single step.
